Formula/mpd.rb

```diff
@@ -92,6 +97,11 @@ def install
     args << "--enable-nfs" if build.with? "libnfs"
     args << "--enable-modplug" if build.with? "libmodplug"
     args << "--enable-pulse" if build.with? "pulseaudio"
+    if build.with? "upnp"
+      args << "--enable-upnp"
+      args << "--enable-expat"
+      args << "--with-expat=/usr/local/opt/expat"
```

Comment:
Indeed, '--with-expat=' option is not neccessary, fixed it.
Thanks for noticing…